corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -119,6 +119,11 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
               {% input_trans module.search_config.search_label.label langs input_name='search_label' input_id='search_label' data_bind="value: searchLabel" %}
             </div>
           </div>
+          <div class="case-search-multimedia-input">
+            <!-- ko stopBinding: true -->
```

Comment:
stopBinding used because the enclosing template with bound to a different object

https://github.com/dimagi/commcare-hq/blob/8469ce862df7561c3cf86205f6a9f49804fdf45e/corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html#L9

and these are to be bound through a different mechanism

https://github.com/dimagi/commcare-hq/blob/5ca2ffe38023b0412c906d1aac1830bf0e4fa032/corehq/apps/app_manager/static/app_manager/js/nav_menu_media.js#L4-L13

Reply:
As with most self-PR comments, this would be a good code comment (maybe without the code references).

Reply:
Alternatively, what would you think of adding a param to `nav_menu_media.html` to optionally include the `stopBindings`? Our custom knockout components do something similar; they have a flag indicating whether they should apply their own bindings or not.

Reply:
I think its better this way so that one can see in the parent template itself that a section of it is not binding along with the parent. If we move it within the nested template, that visibility would go.
Though, I will be happy to make the change to pass a param if it seems better.

Reply:
I think it can go either way, happy to defer to your judgment.